### PR TITLE
[PUBDEV-6540] add definitions for xgb on ubuntu 18.04

### PR DESCRIPTION
**Pull request by honzasterba**

### CODE REVIEW

**Comment by honzasterba** [comment]
@michal-raska this is fun, the download links for jdk8 no longer work, Oracle has changed licence and we should not be using the jdk8 unless we are paying for support, lets switch these to newer jdk?

**Comment by michalkurka** [comment]
Can we do open jdk?

**Comment by michalkurka** [comment]
Java 8 is what enterprise customers are using.

File: scripts/jenkins/groovy/buildConfig.groovy [hunk 1]
```diff
@@ -121,6 +120,11 @@ class BuildConfig {
...
         [name: 'Ubuntu 16.04 OMP', dockerfile: 'xgb/ubuntu/Dockerfile-ubuntu-omp', fromImage: 'harbor.h2o.ai/opsh2oai/h2o-3-xgb-runtime-minimal:ubuntu16', targetName: XGB_TARGET_OMP, nodeLabel: getDefaultNodeLabel()],
         [name: 'Ubuntu 16.04 GPU', dockerfile: 'xgb/ubuntu/Dockerfile-ubuntu-gpu', fromImage: 'nvidia/cuda:8.0-devel-ubuntu16.04', targetName: XGB_TARGET_GPU, nodeLabel: getGPUNodeLabel()],
+      ],
+      'ubuntu18': [
+        [name: 'Ubuntu 18.04 Minimal', dockerfile: 'xgb/ubuntu/Dockerfile-ubuntu-minimal', fromImage: 'ubuntu:18.04', targetName: XGB_TARGET_MINIMAL, nodeLabel: getDefaultNodeLabel()],
+        [name: 'Ubuntu 18.04 OMP', dockerfile: 'xgb/ubuntu/Dockerfile-ubuntu-omp', fromImage: 'harbor.h2o.ai/opsh2oai/h2o-3-xgb-runtime-minimal:ubuntu18', targetName: XGB_TARGET_OMP, nodeLabel: getDefaultNodeLabel()],
+        [name: 'Ubuntu 18.04 GPU', dockerfile: 'xgb/ubuntu/Dockerfile-ubuntu-gpu', fromImage: 'nvidia/cuda:10.0-devel-ubuntu18.04', targetName: XGB_TARGET_GPU, nodeLabel: getGPUNodeLabel()],
```
Comment (michal-raska) [inline scripts/jenkins/groovy/buildConfig.groovy]: Cuda 10 is ok?

Comment (honzasterba) [reply scripts/jenkins/groovy/buildConfig.groovy]: looks good
http://mr-0xc1:8080/view/H2O-3/job/h2o-3-xgb-smoke-pipeline/job/honza%252FPUBDEV-6540/1/

**Pull request merged**